src/index.js

```diff
@@ -0,0 +1,104 @@
+const tempDegree = document.querySelector('#temperatureDegree');
+const landscape = document.querySelector('#landscape');
+const selectedSky = document.querySelector('#skyEmojis');
+const gardenBackground = document.querySelector('#weatherGardenContainer');
+const inputCity = document.querySelector('#inputCity');
+const displayCity = document.getElementById('cityNameDisplay');
+const resetButton = document.getElementById('resetCity');
+
+const state = {
+  tempNumber: 80,
+  skyState: ' ☁️ ☁️ ☀️ ☁️ ☁️ ☀️ ☀️ ☀️ ☁️ ☀️☀️ ☀️ ☁️ ☀️',
+  gardenColor: '#D6FFFF',
+};
+
+// Temperature up and down buttons
+const increaseTemperature = () => {
+  state.tempNumber += 1;
+  tempDegree.textContent = `${state.tempNumber}`;
+  changeTheme();
+};
+
+const decreaseTemperature = () => {
+  state.tempNumber -= 1;
+  tempDegree.textContent = `${state.tempNumber}`;
+  changeTheme();
+};
+
+//Landscape and weather garden background color
+const changeTheme = () => {
+  // if different ranges, change color
+  if (state.tempNumber >= 80) {
+    tempDegree.style.color = '#EA0009';
+    landscape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂🌵__🐍_🏜_🦂';
+  } else if (state.tempNumber >= 70 && state.tempNumber <= 79) {
+    tempDegree.style.color = '#F0940A';
+    landscape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷🌿_☘️🌱_🌻🌷';
+  } else if (state.tempNumber >= 60 && state.tempNumber <= 69) {
+    tempDegree.style.color = '#F4D10A';
+    landscape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃__🛤_🌾🌾🌾_🍃';
+  } else if (state.tempNumber >= 50 && state.tempNumber <= 59) {
+    tempDegree.style.color = '#1F7001';
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲🍂🌲🍁🌲🌲⛄️🍂';
+  } else {
+    tempDegree.style.color = '#256D6C';
+    landscape.textContent =
+      '⛄️⛄️🌲⛄️⛄️🌲⛄️⛄️🌲⛄️🌲⛄️🌲⛄️🌲⛄️⛄️🌲⛄️🌲⛄️';
+  }
+};
+
+//City name reset button
+const updateCity = (event) => {
+  displayCity.textContent = event.target.value;
+};
+
+//reset City
+const resetCity = (event) => {
+  event.target.value = 'Seattle';
```

Comment:
Rather than setting the `event.target.value` to  'Seattle', we want to selected the input box itself, and set the value of that element to "Seattle". This will put 'Seattle' into the input box.